src/store/actions/workStyle.js

```diff
@@ -1,9 +1,29 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    const { title_primary, title_complementary, Cards } = data;
+    const sanitizedCards = Cards.map(card => {
+      return {
+        description: card.description,
+        Title: card.Title,
```

Comment:
you're mixing camelCase and PascalCase ( `Title` instead of `title`) in the object you're returning